src/coreclr/jit/instrsxarch.h

```diff
@@ -611,7 +611,12 @@ INST3(kmovw_msk,         "kmovw",           IUM_WR, PCKFLT(0x91), BAD_CODE,
 INST3(kmovd_msk,         "kmovd",           IUM_WR, PCKDBL(0x91), BAD_CODE,     PCKDBL(0x90),                            INS_TT_NONE,                                          REX_W1       | Encoding_VEX)
 INST3(kmovq_msk,         "kmovq",           IUM_WR, PCKFLT(0x91), BAD_CODE,     PCKFLT(0x90),                            INS_TT_NONE,                                          REX_W1       | Encoding_VEX)
 
-INST3(LAST_K_INSTRUCTION, "LAST_K_INSTRUCTION", IUM_WR, BAD_CODE, BAD_CODE, BAD_CODE, INS_TT_NONE, INS_FLAGS_None)
+INST3(kortestb,          "kortestb",        IUM_WR, BAD_CODE,     BAD_CODE,     PACK3(0x66, 0x0F, 0x98),                 INS_TT_NONE,                                          REX_W0       | Encoding_VEX | Writes_ZF | Writes_CF ) 
```

Comment:
`PACK3(0x66, 0x0F, *)` is effectively `PCKDBL(*)`
`PACK2(0x0F, *)` is effectively `PCKFLT(*)`

Saves a little bit of space and keeps things overall consistent, even if modern SIMD means "more".

Reply:
I'll switch to flag. I'm not sure I see how `Encoding_VEX `or any existing flag can identify `KInstruction`. So, I imagine this means adding `KInstriction` as a flag and updating

```
bool emitter::IsKInstruction(instruction ins)
{
    return (ins >= INS_FIRST_K_INSTRUCTION) && (ins <= INS_LAST_K_INSTRUCTION);
}
```

to


```
bool emitter::IsKInstruction(instruction ins)
{
    insFlags flags = CodeGenInterface::instInfo[ins];
    return (flags & KInstruction) != 0;
}
```

Reply:
Got it. I'm fine with removing the flag if it's better. But I'm going to leave it in there, based on your above comment. Feel free to correct me if you'd rather I take it out

Reply:
Nope leaving it is fine. Just wanted to elaborate on my initial thinking since it wasn't clear :)